### PR TITLE
starting directory for pktvisor automated tests

### PR DESCRIPTION
**Pull request by manrodrigues**

### CODE REVIEW

File: python-test/docs/pktvisor/run_multiple_pktvisors_using_different_ports.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+## Scenario: Run multiple pktvisors using different ports 
+
+## Steps:
+- Provide 1 pktvisor using `docker run --net=host -d ns1labs/pktvisor pktvisord <net>`
```
Comment (ghost) [inline python-test/docs/pktvisor/run_multiple_pktvisors_using_different_ports.md]: provide 1 pktvisor instance

File: python-test/docs/pktvisor/run_multiple_pktvisors_using_different_ports.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+## Scenario: Run multiple pktvisors using different ports
```
Comment (ghost) [inline python-test/docs/pktvisor/run_multiple_pktvisors_using_different_ports.md]: run multiple pktvisor instances

File: python-test/features/steps/pktvisor.py [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+    if port != "default":
+        pkt_command.insert(-1, '-p')
+        pkt_command.insert(-1, port)
+    container = client.containers.run(container_image, name=PKTVISOR_CONTAINER_NAME, detach=True,
+                                      network_mode='host', command=pkt_command)
+    return container.id
+
+
+
+@step("run pktvisor on port {pkt_port}")
```
Comment (ghost) [inline python-test/features/steps/pktvisor.py]: pktvisor instance

File: python-test/docs/pktvisor/run_multiple_pktvisors_using_different_ports.md (hunk 1 above)
Comment (manrodrigues) [reply python-test/docs/pktvisor/run_multiple_pktvisors_using_different_ports.md]: ok

File: python-test/docs/pktvisor/run_multiple_pktvisors_using_different_ports.md (hunk 2 above)
Comment (manrodrigues) [reply python-test/docs/pktvisor/run_multiple_pktvisors_using_different_ports.md]: ok

File: python-test/features/steps/pktvisor.py (hunk 3 above)
Comment (manrodrigues) [reply python-test/features/steps/pktvisor.py]: ok

File: automated_tests/README.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+```
+Feature: pktvisor tests # features/pktvisor.feature:1
+
+  Scenario: pktvisor bootstrap                         # features/pktvisor.feature:3
+    When run pktvisor instance on port default         # features/steps/pktvisor.py:33 0.150s
+    Then the pktvisor container status must be running # features/steps/pktvisor.py:39 0.007s
+    And pktvisor API must be enabled                   # features/steps/pktvisor.py:75 1.123s
+
+  Scenario: run multiple pktvisors using different ports  # features/pktvisor.feature:8
```
Comment (ghost) [inline automated_tests/README.md]: multiple pktvisor instances

File: automated_tests/README.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+    When run pktvisor instance on port default            # features/steps/pktvisor.py:33 0.156s
```
Comment (leoparente) [inline automated_tests/README.md:56]: When run **a** pktvisor instance

File: automated_tests/features/pktvisor.feature [hunk 6]
```diff
@@ -0,0 +1,19 @@
+Feature: pktvisor tests
+
+Scenario: pktvisor bootstrap
+  When run pktvisor instance on port default
+  Then the pktvisor container status must be running
+    And pktvisor API must be enabled
+
+Scenario: run multiple pktvisors using different ports
```
Comment (ghost) [inline automated_tests/features/pktvisor.feature]: multiple pktvisor instances

**Review by ghost — APPROVED** [review]

**Pull request merged**